### DISCUSSION TITLE
mod_fcgid module for Apache

### DISCUSSION THREAD

**Discussion by jirisGitHub** [termux/termux-packages#11925] [General]
Hi!

On my Android 9 device I installed Apache2 server in Termux.
I need to enable DSO (Dynamic Shared Object) mod_fcgid module to run perl fcgi scripts.
Unfortunately this module is not present in /libexec/apache2 directory so I cannot load it 
in my httpd.conf configuration LoadModule section.
Would it be possible to get somehow the binary mod_fcgid.so for armv7l architecture?
Thank you very much.

Jiri Schuster
Czech Republic

**Comment by 2096779623** [comment]
try to build this manually.

https://github.com/apache/httpd-mod_fcgid
